tests/openvino/tools/calibrate.py

```diff
@@ -71,7 +72,9 @@ def parse_args():
 
 class CustomJSONEncoder(json.JSONEncoder):
     def default(self, o):
-        if isinstance(o, (TargetDevice, ModelType, QuantizationPreset, OverflowFix, StatisticsType, AggregatorType)):
+        if isinstance(
+            o, (TargetDevice, ModelType, QuantizationPreset, OverflowFix, StatisticsType, AggregatorType, DropType)
+        ):
```

Comment:
Wouldn't be better to use `hasattr(o, 'value')` here instead of extending the list of classes?

Reply:
done